crates/nu-command/tests/commands/ls.rs

```diff
@@ -430,6 +430,28 @@ fn lists_files_including_starting_with_dot() {
     })
 }
 
+#[test]
+fn lists_regular_files_using_question_mark_wildcard_asdsad() {
```

Comment:
Ups, that need to be removed, it were just some experimental code. 

Will wait until I get some input on how to setup the automatic tests before changing this.

Reply:
Fixed and pushed.